nsv13/code/modules/jobs/job_types/fighter_pilot.dm

```diff
@@ -17,7 +17,8 @@
 	access = list(ACCESS_MAINT_TUNNELS, ACCESS_MUNITIONS, ACCESS_MUNITIONS_STORAGE, ACCESS_FIGHTER) //temp
 	minimal_access = list(ACCESS_MAINT_TUNNELS, ACCESS_MUNITIONS, ACCESS_FIGHTER) //temp
 	paycheck = PAYCHECK_MEDIUM
-	paycheck_department = ACCOUNT_CAR
+	paycheck_department = ACCOUNT_MUN
+	mind_traits = list(TRAIT_MUNITIONS_METABOLISM)
```

Comment:
i dont think pilots should get this trait, i mean technically they are apart of muni but they're not MTs

Reply:
All or nothing.